src/hooks/useArNSContract/useArNSContract.tsx

```diff
@@ -8,35 +8,25 @@ const ARNS_SOURCE_CONTRACT_ID = 'bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U';
 export default function useArNSContract() {
   // eslint-disable-next-line
   const [{}, dispatch] = useStateValue();
-  const [sendingContractState, setSendingContractState] = useState(false);
 
   useEffect(() => {
     dispatchNewContractState();
-  });
+  }, []); // eslint-disable-line
 
   async function dispatchNewContractState(): Promise<void> {
     try {
       const localProvider = new LocalFileSystemDataProvider();
-      if (sendingContractState) {
-        return;
-      }
-      setSendingContractState(true);
-
```

Comment:
i'd prefer we keep this as it'll become more relevant when we start fetching state